src/test/shell/bazel/bazel_symlink_test.sh

```diff
@@ -370,25 +357,35 @@ def _a_impl(ctx):
     )
     return DefaultInfo(files = depset([output]))
 
-a = rule(implementation = _a_impl, attrs = {"link_target": attr.string()})
+a = rule(
+    implementation = _a_impl, 
```

Comment:
FYI: I will remove the trailing space from this line as part of importing this change. You can delete that yourself if you prefer.

Reply:
Thanks!